wannierberri/__system_w90.py

```diff
@@ -65,6 +65,7 @@ def __init__(self,seedname="wannier90",
         self.iRvec,self.Ndegen=self.wigner_seitz(chk.mp_grid)
         self.nRvec0=len(self.iRvec)
         self.num_wann=chk.num_wann
+        self.wannier_centres = chk.wannier_centres.dot(np.linalg.inv(self.real_lattice))
```

Comment:
It is better to distinguish the naming of variables `wannier_centres_cart` and `wannier_centres_reduced` to avoid confusion.

Reply:
Done